configurations/postgres-reactive/src/main/java/io/micronaut/configuration/postgres/reactive/health/PgPoolHealthIndicator.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright 2017-2018 original authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.micronaut.configuration.postgres.reactive.health;
+
+import io.micronaut.context.annotation.Requires;
+import io.micronaut.health.HealthStatus;
+import io.micronaut.management.health.indicator.HealthIndicator;
+import io.micronaut.management.health.indicator.HealthResult;
+import io.reactiverse.reactivex.pgclient.PgPool;
+import io.reactiverse.reactivex.pgclient.PgRowSet;
+import io.reactivex.Flowable;
+import io.reactivex.Single;
+import io.reactivex.schedulers.Schedulers;
+import org.reactivestreams.Publisher;
+
+import javax.inject.Singleton;
+
+/**
+ * A Health indicator for reactive Postgres client.
+ *
+ * @author puneetbehl
+ * @since 1.0
+ */
+@Requires(classes = HealthIndicator.class)
+@Singleton
+public class PgPoolHealthIndicator implements HealthIndicator {
```

Comment:
There needs to be a way to disable this

Reply:
I looked through HealthIndicator implementations for other configurations modules. Could you please point to me to some example implementations?